internal/aws/ses.go

```diff
@@ -0,0 +1,86 @@
+package aws
+
+import (
+	"log"
+
+	"github.com/RobinHoodArmyHQ/robin-api/models"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
+)
+
+// SendEmailSES sends email to specified email IDs
+func SendEmailSES(messageBody string, subject string, fromEmail string, recipient models.Recipient) (err error) {
+	// create new AWS session
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String("us-east-1")},
```

Comment:
should keep this in `config.yaml` and can should also modify docker compose to setup ses on local.
https://docs.localstack.cloud/user-guide/aws/ses/